docs/connecting-data/connecting-to-apis-and-external-data/virustotal-api.md

```diff
@@ -5,5 +5,105 @@ description: How to Connect DataDistillr to the VirusTotal API
 # Connecting to VirusTotal
 
 ## First Steps with VirusTotal
+Set up an account with [VirusTotal](https://www.virustotal.com/gui/join-us){target=_blank}.
 
-Details coming soon!
\ No newline at end of file
+???+ cost "Cost"
+
+    There is a free version and a [paid](https://www.virustotal.com/gui/contact-us/premium-services){target=_blank} version. 
+
+
+???+ rlimit "Rate Limits"
+
+    With the free version the API is [rate limited](https://developers.virustotal.com/reference/public-vs-premium-api){target=_blank}.
+
+## How to Connect DataDistillr to VirusTotal
+To set up a data source connection for VirusTotal, you will need to have:
+
+- A unique [name](#name) for your data source connection to be used in queries.
+- The [API Key](#api-key) generated by VirusTotal.
+
+
+
+### Data Source Form
+
+To locate the VirusTotal form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.&#x20;
+
+![Select API from the available choices][image-5]
+
+On the API screen, select VirusTotal from list of API forms as shown in the image below.
+
+![Select VirusTotal API from available choices][image-6]
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the VirusTotal API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+![VirusTotal Form][image-1]
+
+### Name
+
+Enter any name that will help you recognize this data source from within your query window. &#x20;
+
+!!! info "Acceptable characters include"
+
+    * lowercase alphanumeric characters
+    * underscores
+
+### API Key
+
+=== "1. Find your API Key"
+
+    From your VirusTotal home page, open the menu in the to right hand corner and click on "API Keys" tab in the menu
+    ![Sign in][image-8]
```

Comment:
This picture is very hard to see. Please just show the top right corner of the page with the dropdown so it's easier to see.